katello/hooks/boot/13-hiera.rb

```diff
@@ -0,0 +1,11 @@
+# TODO automatically get the tuning sizes - standard doesn't exist and only
+# loads base. The rest maps to config/foreman.hiera/tuning/sizes/$size.yaml
```

Comment:
Can we capture this TODO in a Redmine issue? Also the text is already out of date :) Should be "default doesn't exist"